prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py

```diff
@@ -171,6 +171,9 @@ def omen_replicate_from_tx(
             auto_deposit=auto_deposit,
             collateral_token_address=sDaiContract().address,
         )
+        market_address = ChecksumAddress(
+            created_market.market_event.fixedProductMarketMaker
```

Comment:
You can use `created_market.market_event.fixed_product_market_maker_checksummed` directly here.

Btw if you need to initialise ChecksumAddress, use `Web3.to_checksum_address(...)`, because it's not just a type, address indeed looks different afterwards:

```
>>> addr
'0xab16d643ba051c11962da645f74632d3130c81e2'
>>> Web3.to_checksum_address(addr)
'0xAB16D643bA051C11962DA645f74632d3130c81E2'
>>> ChecksumAddress(addr)
'0xab16d643ba051c11962da645f74632d3130c81e2'
```

Reply:
Ah thanks!

Reply:
One more neat pick that surrounding it in ChecksumAddress is not required anymore, as `fixed_product_market_maker_checksummed` is of ChecksumAddress type already 🙏